Add mcp automated testing blog

ravarora2 opened the pull request:
Add mcp automated testing blog
[Automated MCP Testing_ Using Composable Goose Recipes to Validate Tool Metadata _ codename goose.pdf](https://github.com/user-attachments/files/21724440/Automated.MCP.Testing_.Using.Composable.Goose.Recipes.to.Validate.Tool.Metadata._.codename.goose.pdf)

blackgirlbytes commented:
Hi there! Thanks for your contribution. It looks like you're missing metadata tags for your blog post. Place something like this at the bottom of your blog post. Replace the current values with values that are relevant to your post

```html
<head>
  <meta property="og:title" content="MCP UI, bringing the browser into the agent" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="https://block.github.io/goose/blog/2025/08/11/mcp-ui-post-browser-world" />
  <meta property="og:description" content="A first look at a UI for agents built on the proposed MCP-UI extension" />
  <meta property="og:image" content="https://block.github.io/goose/assets/images/mcp-ui-shot-1b80ebfab25d885a8ead1ca24bb6cf13.png" />
  <meta name="twitter:card" content="summary_large_image" />
  <meta property="twitter:domain" content="block.github.io/goose" />
  <meta name="twitter:title" content="A first look at a UI for agents built on the proposed MCP-UI extension" />
  <meta name="twitter:description" content="A first look at a UI for agents built on the proposed MCP-UI extension" />
  <meta name="twitter:image" content="https://block.github.io/goose/assets/images/mcp-ui-shot-1b80ebfab25d885a8ead1ca24bb6cf13.png" />
</head>
```

ravarora2 commented:
> Hi there! Thanks for your contribution. It looks like you're missing metadata tags for your blog post. Place something like this at the bottom of your blog post. Replace the current values with values that are relevant to your post
> 
> ```
> <head>
>   <meta property="og:title" content="MCP UI, bringing the browser into the agent" />
>   <meta property="og:type" content="article" />
>   <meta property="og:url" content="https://block.github.io/goose/blog/2025/08/11/mcp-ui-post-browser-world" />
>   <meta property="og:description" content="A first look at a UI for agents built on the proposed MCP-UI extension" />
>   <meta property="og:image" content="https://block.github.io/goose/assets/images/mcp-ui-shot-1b80ebfab25d885a8ead1ca24bb6cf13.png" />
>   <meta name="twitter:card" content="summary_large_image" />
>   <meta property="twitter:domain" content="block.github.io/goose" />
>   <meta name="twitter:title" content="A first look at a UI for agents built on the proposed MCP-UI extension" />
>   <meta name="twitter:description" content="A first look at a UI for agents built on the proposed MCP-UI extension" />
>   <meta name="twitter:image" content="https://block.github.io/goose/assets/images/mcp-ui-shot-1b80ebfab25d885a8ead1ca24bb6cf13.png" />
> </head>
> ```

Added and also replaced the title image.

Thanks
